code/modules/modular_computers/file_system/programs/generic/map.dm

```diff
@@ -45,6 +45,16 @@
 		list("d"="Hangar", "c"=HOLOMAP_AREACOLOR_HANGAR),
 	)
 
+	data["pois"] = list()
+	for(var/obj/effect/landmark/minimap_poi/poi in SSholomap.pois)
```

Comment:
If the pois list contains only those, skip the useless typecheck of for:

```suggestion
	for(var/obj/effect/landmark/minimap_poi/poi as anything in SSholomap.pois)
```

Reply:
done